src/App.jsx

```diff
@@ -1,35 +1,38 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React from "react";
+import { useEffect, useState } from "react";
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import './App.css';
+import Header from "./Header";
+import Nav from "./Nav";
+import Home from "./Home";
+import Articles from "./Articles";
+import { getArticles } from './utils/api';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
```

Comment:
Have a think about whether this state needs to be in App, or can it be moved lower?